src/ruby.rs

```diff
@@ -740,6 +740,25 @@ extern {
     ///
     //+ c-func: variable.c `const char *rb_obj_classname(VALUE)`
     pub fn rb_obj_classname(obj: VALUE) -> *const c_char;
+
+    /// Returns a C boolean (zero if false, non-zero if true) indicated whether yield would
+    /// execute a block in the current method.
+    ///
+    /// This is the equivalent of `Kernel#block_given?`.
+    ///
+    /// # Safety
+    ///
+    /// No known issues
+    ///
+    /// # Defined In
+    ///
+    /// * **2.3:** [ruby.h](https://github.com/ruby/ruby/blob/v2_3_7/include/ruby/ruby.h#L1830)
+    /// * **2.4:** [ruby.h](https://github.com/ruby/ruby/blob/v2_4_4/include/ruby/ruby.h#L1850)
+    /// * **2.5:**
+    ///     [ruby.h](https://github.com/ruby/ruby/blob/v2_5_1/include/ruby/ruby.h#L1877) |
+    ///     [eval.c](https://github.com/ruby/ruby/blob/v2_5_1/eval.c#L834-L843)
+    /// * **2.6:** [ruby.h](https://github.com/ruby/ruby/blob/v2_6_0_preview2/include/ruby/ruby.h#L1894)
```

Comment:
@wagenet can you help @zvkemp with the format here?

Reply:
The whole `Defined In` section should be replaced with:

```
\\+ c-func eval.c `int rb_block_given_p(void)`
````